network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/listing_block.html

```diff
@@ -2,24 +2,26 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 
 {% block block_content %}
-	<div class="tw-row tw-py-4">
-		{% for block in self.cards %}
-          {% with card=block %}
-            {% image card.image fill-350x197 as img %}
-            <div class="tw-px-4 medium:tw-w-1/2 tw-mb-5">
-              <div class="card-regular tw-h-full tw-flex tw-flex-col">
-                <img src="{{ img.url }}" alt="{{ card.alt_text }}" />
-                <div class="tw-flex tw-flex-1">
-                  <div class="tw-bg-white tw-relative tw-py-4 tw-w-full tw-flex tw-flex-col">
-                    {% if card.link %}<a href="{% pageurl card.link %}">{% endif %}
-                        <h3 class="tw-h3-heading tw-mb-2">{{ card.title }}</h3>
-                    {% if card.link %}</a>{% endif %}
-                    {{ card.body|richtext }}
-                  </div>
-                </div>
+  <div class="tw-row tw-py-4">
+    {% for block in self.cards %}
+      {% with card=block %}
+        {% image card.image fill-350x197 as img %}
+        <div class="tw-px-4 medium:tw-w-1/2 tw-mb-5 group">
+          <div class="card-regular tw-h-full tw-flex tw-flex-col">
+            <img src="{{ img.url }}" alt="{{ card.alt_text }}"/>
+            <div class="tw-flex tw-flex-1">
+              <div class="tw-bg-white tw-relative tw-py-4 tw-w-full tw-flex tw-flex-col">
+                <h3 class="tw-h3-heading tw-mb-2 grou">
```

Comment:
I think there is a leftover in here. 

```suggestion
                <h3 class="tw-h3-heading tw-mb-2">
```

Reply:
"I am grou" 

haha oops! Was testing some styling using the group class originally 😅. Thanks for catching this